lib/aruba/generators/script_file.rb

```diff
@@ -0,0 +1,79 @@
+# Aruba
+module Aruba
+  # ScriptFile
+  class ScriptFile
+
```

Comment:
please remove empty lines. Otherwise `rubocop` will complain and the build will fail. ;-)